modules/manage/pages/manage-settings/configure-alerts.adoc

```diff
@@ -202,17 +202,30 @@ The size of the change history may need to be increased.
 For information, on establishing change-history size, see xref:rest-api:rest-bucket-create.adoc[Creating and Editing Buckets].
 | `history_size_warning`
 
-| Low Indexer Residence Percentage
+| Approaching Indexer low resident percentage
 | Warns that the Index Service is, on a given node, occupying a percentage of available memory that is below an established threshold, the default for which is `10`.
 | `indexer_low_resident_percentage`
 
 a| [#memcached-alert]
 Memcached connection threshold exceeded.
 | Trigger an alert if the number of `system` or `user` connections used by the data service exceeds a configurable percentage of the available connections{blank}xref:#memcached-alert-foonote[^1^].
-For information on setting the `memcached` alert thresholds, see xref:rest-api:rest-cluster-email-notifications.adoc#setting-memcache-alert-threshold[Setting alerts].
+For information about setting the `memcached` alert thresholds, see xref:rest-api:rest-cluster-email-notifications.adoc#setting-memcache-alert-threshold[Setting alerts].
 | `memcached_connections`
 
+| Rebalance stage appears stuck
+| An ongoing KV or index rebalance has not made progress during the timeout period set by the `stuckRebalanceThresholdIndex` and `stuckRebalanceThresholdKV` alert limits.
+The default value for the timeout period is 1800 seconds (30 minutes).
+| `stuck_rebalance`
 
+| Disk usage is within 10% of maximum for data service mutations
+| The used disk space on the a filesystem containing the Data Service storage path is within 10% of the configured limit. 
+This limit is set either through the Advanced Data Settings in the Couchbase Server Web Console, or by using the `/settings/resourceManagement` REST API endpoint.
+See xref:learn:buckets-memory-and-storage/storage-settings.adoc#filesystem-free-space-and-usage-limits[Filesystem Free Space and Usage Limits] for more information.
+| `disk_guardrail`
```

Comment:
This is a minor point, but I'm confused as to where this name `disk_guardrail` is coming from -- since that word isn't used anywhere else in the documentation or the REST API in reference to this feature.

cc @Peter-Searby 

% curl -s -X GET http://localhost:8091/settings/resourceManagement -u Administrator:password | jq .
{
  "diskUsage": {
    "enabled": false,
    "maximum": 85
  }
}

Note:  I do appreciate that this feature is being documented thoroughly since the resource name is pretty generic.